pkg/attribute/attribute.go

```diff
@@ -52,23 +52,25 @@ type description struct {
 	typ          reflect.Type
 	defaultValue interface{}
 	doc          string
-	checker      func(i interface{}) error
+	checker      func(value interface{}, name string) error
 }
 
 // Int declares an attribute of int-typed in Dictionary d.
 func (d Dictionary) Int(name string, value interface{}, doc string, checker func(int) error) Dictionary {
-	interfaceChecker := func(v interface{}) error {
-		if intValue, ok := v.(int); ok {
+	interfaceChecker := func(value interface{}, name string) error {
```

Comment:
It seems we don't have to pass `name` to `interfaceChecker` because `name` is included in the closure of `interfaceChecker`. Anyway, we can fix it in another pull request.

Reply:
We may need to pass `name` to `interfaceChecker`, because the `name` (for example, `model.*`) when calling `Dictionary.Int` may be different from the name when calling `checker(value, name)` (for example `model.n_class`).